docs/running-locally.md

```diff
@@ -0,0 +1,85 @@
+# How to setup your dev environment
+## Pre-requisites:
+
+* Dep. `brew install dep`
```

Comment:
On a different note, should we provide a dev container images that folks can use if they want so they don't have to install all these tools?

Reply:
someday, but not today